crates/kotlin-ffi/src/lib.rs

```diff
@@ -34,43 +34,43 @@ pub struct AccountClientConfig {
 }
 
 #[derive(uniffi::Record)]
-pub struct Transaction {
+pub struct FFITransaction {
```

Comment:
Why do we need to change the names of these structs?

Reply:
same as above